README.md

```diff
@@ -8,106 +7,58 @@ protocol using the
 [Scala](https://www.scala-lang.org/) 
 programming language.
 
-```scala
-constellation
-└── src
-    ├── it                                   <─── continuous integration
-        ├── ClusterDebug.scala
-        ├── ...
-        └── UploadChain.scala
-    ├── main/scala/org/constellation         <─── protocol implementation
-        ├── consensus
-            ├── Consensus.scala
-            ├── EdgeProcessor.scala
-            └── ...
-        ├── crypto
-            ├── Base58.scala
-            ├── KeyUtils.scala
-            └── Wallet.scala
-        ├── p2p
-            └── ...
-        ├── primitives
-            ├── ...
-            └── TransactionValidation.scala
-        ├── serializer
-        ├── util
-            ├── API.scala
-            ├── ConstellationNode.scala
-            ├── Data.scala
-            └── LevelDB.scala
-    └── test/scala/org/constellation          <─── unit tests
-        ├── app
-        ├── cluster
-        ├── consensus
-        ├── p2p
-        ├── rpc
-        ├── ...
-        └── UtilityTest.scala
-└── ui
-```
-
-(Not shown in the above table are things like fixtures, shell- and sbt-scripts or markup-, package-, scala-project and config-files that are used for project building, deploying, docker, kubernetes, network setup, unit tests, continuous integration, and so on.)
-
-## :computer: Building for development
-### On Linux and Mac
-1. Check out the repository 
-```bash
-git clone git@github.com:Constellation-Labs/constellation.git
-```
-2. From root directory `constellation`, run 
-```bash
-./build.sh
-```
-or optionally (to connect to other host)
-```bash
-./build.sh seedhost:port
-```
-3. Interact with app via the API 
-By default this is 
-http://localhost:9000. 
-Look at the file _API.scala_ for endpoints.
+Visit the [/constellation/docs](https://github.com/Constellation-Labs/constellation/docs/) for an explainer of the protocol archetecture or an extended overview of the 
+[project structure](https://github.com/Constellation-Labs/constellation/docs/directory-tree/). Note that the developer branch might be behind the master branch by a few weeks.
```

Comment:
Do you mean the master branch might be behind the developer branch by a few weeks?

Reply:
Yes, lol, good catch.